.github/workflows/ci.yml

```
@@ -18,5 +18,21 @@ jobs:
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v6
+
+      - name: Set up Go
+        uses: actions/setup-go@v6
+        with:
+          go-version: "1.27.1"
+
+      - name: Check formatting
+        run: test -z $(go fmt ./...)
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:9: Quote this to prevent word splitting

(shellcheck)